README.md

```diff
@@ -629,6 +629,35 @@ Other Style Guides
   const y = function a() {};
   ```
 
+- [7.12](#7.12) <a name="7.12"></a> Never mutate parameters.
+
+  > Why? Overwriting parameters can lead to unexpected behavior, especially when accessing the `arguments` object. Manipulating objects passed in as parameters can cause unwanted variable side effects in the original caller.
+
+  eslint rules: [`no-param-reassign`](http://eslint.org/docs/rules/no-param-reassign.html).
+
+  ```javascript
+  // bad
+  function f(a){
+    a = 1;
+  }
+  function f(a){
+    if (!a) { a = 1; }
+  }
+  function f(obj){
```

Comment:
i can tell these are meant to be 3 separate examples, but to avoid looking like we're redeclaring functions, let's use different names for them